.laminas-ci/pre-install.sh

```diff
@@ -4,4 +4,10 @@ WORKING_DIRECTORY=$2
 JOB=$3
 PHP_VERSION=$(echo "${JOB}" | jq -r '.php')
 
-apt install -y php8.1-ldap || exit 1
+apt-get install -y iptables conntrack || exit 1
+iptables-restore --verbose --test ./.ci/iptables.rules
```

Comment:
This doesn't work, the rules are not accepted, no matter what I tried:

```
iptables-restore v1.8.4 (legacy): iptables-restore: unable to initialize table 'filter'
# Generated by iptables-save v1.4.21 on Fri Sep 29 15:37:54 2017

Error occurred at line: 2
```

I tested the rule on another system and it was successful, so my guess is that this is some kind of docker issue that I don't understand. I extracted the rules to a different file, you for the sake of testing, because nothing else worked....

All `ReconnectTest` issues are caused by this problem.